platform.md

```diff
@@ -104,16 +107,12 @@ Platform API versions:
 #### CNB Terminology
 A **buildpack** refers to software compliant with the [Buildpack Interface Specification](buildpack.md).
 
-A **stack** is a contract, implemented by a **build image** and **run image**, that guarantees properties of the **build environment** and **app image**.
-
-A **stack ID** uniquely identifies a particular **stack**.
+A **base image** is an OCI image containing the base, or initial set of layers, for other images.
```

Comment:
Shouldn't the two lines below reference this definition? I.e., shouldn't they both say "… is an OCI image that ~provides~ *serves as* the base *image* of the …"?

Reply:
Thank you! I updated in 4f4aa24